django-stubs/contrib/gis/geos/geometry.pyi

```diff
@@ -12,127 +12,127 @@ class GEOSGeometryBase(GEOSBase):
     has_cs: bool = ...
     __class__: Any = ...
     def __init__(self, ptr: Any, cls: Any) -> None: ...
-    def __copy__(self): ...
-    def __deepcopy__(self, memodict: Any): ...
+    def __copy__(self) -> Any: ...
```

Comment:
Can be turned into `(self: _T) -> _T` in this PR or later (probably later, since we have more `__copy__` methods to fix)

And the same for `__deepcopy__`